src/api/search/server.js

```diff
@@ -0,0 +1,11 @@
+const { Satellite } = require('@senecacdot/satellite');
+
+// const health = require('./src/routes/health');
+const query = require('./src/routes/query');
+
+const service = new Satellite();
+// service.router.use('/health', health);
```

Comment:
A generic healthcheck endpoint gets created, and if you want to add something to also call-out to ES and see if it's up, that would be a logical thing to do here.

Reply:
Agreed, going to add.

Reply:
I was wondering, should we bake the /health check into Satellite, or have each micro-service implement it's own health route?